src/libraries/System.Private.CoreLib/src/System/Half.cs

```diff
@@ -539,64 +539,311 @@ public bool TryFormat(Span<char> destination, out int charsWritten, [StringSynta
             return Number.TryFormatHalf(this, format, NumberFormatInfo.GetInstance(provider), destination, out charsWritten);
         }
 
-        // -----------------------Start of to-half conversions-------------------------
+        //
+        // Explicit Convert To Half
```

Comment:
Why does Half have all these converters in this PR? Is there something special about the type, or was it just missing things that the other types already had?

Reply:
The converters weren't previously defined for it. With this functionality being added conversions need to be defined in order for the functionality to work as expected.